servers/rendering/renderer_canvas_cull.h

```diff
@@ -34,6 +34,8 @@
 #include "core/templates/paged_allocator.h"
 #include "renderer_compositor.h"
 #include "renderer_viewport.h"
+#include "rendering_server_globals.h"
+#include "scene/main/viewport.h"
```

Comment:
Servers shouldn't have dependencies on `scene`, see #53295. So this system might need to be registered in `DisplayServer` I guess?

Though it seems weird that window management ends up in `RendererCanvasCull`, but I'm not familiar with this area.